pydoctor/test/test_packages.py

```diff
@@ -89,4 +89,17 @@ def test_package_module_name_clash() -> None:
     """
     system = processPackage('package_module_name_clash')
     pack = system.allobjects['package_module_name_clash.pack']
-    assert 'package' == pack.contents.popitem()[0]
\ No newline at end of file
+    assert 'package' == pack.contents.popitem()[0]
+
+def test_reparented_module() -> None:
+    """
+    Reparented modules does not crash pydoctor, and are correctly reparented like any other objects.
+    """
```

Comment:
Minor comment.

Rather than talking about the crash, I think that is best to describe the
expected behaviour.

And maybe also assert that that original name is no longer availble?
Or reparented is just an alias and both the original and the new names are available?

```suggestion
    """
    A module that is imported in a package as a different name and exported
    in that package under the new name via C{__all__} is presented using the
    new name.
    """
```